src/Faker/Provider/Image.php

```diff
@@ -28,41 +26,55 @@ class Image extends Base
      *
      * @return string
      */
-    public static function imageUrl($width = 640, $height = 480, $category = null, $randomize = true, $word = null, $gray = false)
-    {
-        $baseUrl = "https://lorempixel.com/";
-        $url = "{$width}/{$height}/";
+    public static function imageUrl(
+        int $width = 640,
+        int $height = 480,
+        ?string $category = null,
+        bool $randomize = true,
+        ?string $word = null,
+        bool $gray = false
+    ): string {
+        $size = sprintf('%sx%s.png', $width, $height);
```

Comment:
```suggestion
        $size = sprintf('%dx%d.png', $width, $height);
```
`%d` represents a signed integer and better matches the variable types